src/hypothesis/internal/compat.py

```diff
@@ -478,3 +478,10 @@ def implements_iterator(it):
     # the x flag because it's Python 2, so there are no places where this can
     # be thrown.
     FileExistsError = None
+
+
+if PY2:
```

Comment:
The only other place it's used is in statistics.py, where it's explicitly converted to an int anyway and didn't seem worth tinkering with (it's matched with a ceil, and then I'd have to add ceil to compat.py)